operator/pkg/ciliumidentity/controller_test.go

```diff
@@ -437,7 +437,9 @@ func TestUpdateUsedCIDIsReverted(t *testing.T) {
 	ev.Done(nil)
 }
 
-func TestDeleteUsedCIDIsRecreated(t *testing.T) {
+// TODO: Re-enable it once flakes are fixed.
+// Ref: https://github.com/cilium/cilium/issues/35135
+func testDeleteUsedCIDIsRecreated(t *testing.T) {
```

Comment:
We can use

```go
t.Skip("test disabled, see https://github.com/cilium/cilium/issues/35135")
```

instead, doing that should also make the linter happy.

Reply:
Done.